src/MapBox.js

```diff
@@ -1,23 +1,276 @@
-// src/CardList.js
+// src/MapBox.js
 
-import React from 'react';
+/*******************************************************************************************************************************
+ * File : MapBox.js
+ * In this file, Mapbox component is implemented with react hook.
+ * Markers and lines can be displayed in Mapbox map with props.map.marker.coordinates / props.map.line.coordinates
+ * Author : Jeesub Kim
+ *******************************************************************************************************************************/
+import React, {useEffect, useState, useRef} from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
-const Wrapper = styled.div``;
+// import Box from '@material-ui/core';
```

Comment:
Please remove this unused code.

Reply:
It was supposed be used later, but let me remove